tests/internal/params/vars.go

```diff
@@ -1,9 +1,17 @@
 package params
 
+import "github.com/golang/glog"
+
 // PrivilegedNSLabels represents privileged labels.
-var PrivilegedNSLabels = map[string]string{
-	"pod-security.kubernetes.io/audit":               "privileged",
-	"pod-security.kubernetes.io/enforce":             "privileged",
-	"pod-security.kubernetes.io/warn":                "privileged",
-	"security.openshift.io/scc.podSecurityLabelSync": "false",
-}
+var (
+	PrivilegedNSLabels = map[string]string{
+		"pod-security.kubernetes.io/audit":               "privileged",
+		"pod-security.kubernetes.io/enforce":             "privileged",
+		"pod-security.kubernetes.io/warn":                "privileged",
+		"security.openshift.io/scc.podSecurityLabelSync": "false",
+	}
+	// LogLevel is the verbosity for global tools.
+	LogLevel glog.Level = 90
+	// OpenshiftLoggingNamespace is the namespace where logging pods are.
+	OpenshiftLoggingNamespace = "openshift-logging"
```

Comment:
This parameter has to be added to https://github.com/openshift-kni/eco-gotests/blob/60d66789c0a6707c226d20d344dc1618ccb368ba/tests/internal/config/config.go#L32 instead of hardcode.

Reply:
fixed